packages/native-stack/src/index.tsx

```diff
@@ -8,6 +8,11 @@ export { default as createNativeStackNavigator } from './navigators/createNative
  */
 export { default as NativeStackView } from './views/NativeStackView';
 
+/**
+ * Contexts
+ */
+export { RetainContext } from './views/RetainContext';
```

Comment:
let me know if you want this to go into a `/contexts/` directory, or somewhere else